dpnp/fft/dpnp_utils_fft.py

```diff
@@ -66,6 +69,64 @@ def _check_norm(norm):
         )
 
 
+# TODO: c2r keyword is place holder for irfftn
+def _cook_nd_args(a, s=None, axes=None, c2r=False):
+    if axes is not None:
+        # validate axes is a sequence and
+        # each axis is an integer within the range
+        normalize_axis_tuple(list(set(axes)), a.ndim, "axes")
+
+    if s is None:
+        shapeless = True
+        if axes is None:
+            s = list(a.shape)
+        else:
+            s = numpy.take(a.shape, axes)
+    else:
+        shapeless = False
+        try:
+            iter(s)
+        except Exception as e:
+            raise TypeError(
+                "`s` must be `None` or a sequence of integers."
+            ) from e
+
+        for s_i in s:
+            if s_i is not None and not isinstance(s_i, int):
+                raise TypeError("`s` must be `None` or a sequence of integers.")
+
+    for s_i in s:
+        if s_i is not None and s_i < 1 and s_i != -1:
+            raise ValueError(
+                f"Invalid number of FFT data points ({s_i}) specified."
+            )
+
+    if axes is None:
+        # TODO: uncomment the checkpoint
```

Comment:
It is still supported in NumPy 2.0 but it is deprecated. In future versions, it will raise an Error. So, I added TODO for dpnp to raise Error in future too.

Reply:
I'd propose to add more details in the comment and to state that it s still supported in NumPy 2.0 but deprecated and will be removed in the future version.

Reply:
But probably it also make sense to be align with future NumPy behavior already now.
I mean if the support of this is deprecated in NumPy 2.0. I believe we also should not support that, even in older numpy version the support is present.